README.md

```diff
@@ -43,7 +43,11 @@ to safely remove the Node object once the node is removed from the cloud provide
 ```
 Usage of cloud-lifecycle-controller:
   -cloud string
-        Cloud provider to use [aws, azure, ...]
+        Cloud provider to use (aws, azure, gcs, ...)
+  -cloud-config string
+        Path to cloud provider config file
+  -cloud-zone string
+        Cloud zone (us-west2, us-central, ...)
```

Comment:
in AWS, will this be an aws region `us-east-1` or an availability zone `us-east-1b`?

Reply:
That's a good question. I merely ran the command to get the usage and update the readme. I believe it's meant to be the region but it's not currently used so hard to say for certain. Since it's not currently used it's probably better to pull it out and add it back in when/if we need it.

Reply:
Ok, I removed the flag.